trompace/client/client_itemlist.py

```diff
@@ -139,12 +136,158 @@ def merge_listitem_item_nodes(listitem_id: str, item_id: str):
                                                     listitem_id=listitem_id,
                                                     item_id=item_id)
 
-    resp = trompace.connection.submit_query(mutation)
+    resp = submit_query(mutation)
     result = resp.get("data", {}).get("MergeListItemItem")
 
     if not result:
-        # Raise Error
-        pass
+        raise QueryException(resp['errors'])
+
+
+def update_listitem_position(listitem_id: str, position: int):
+    """
+    """
+    mutation = mutations_itemlist.mutation_update_listitem(identifier=listitem_id,
+                                                           position=position)
+    resp = submit_query(mutation)
+    result = resp.get("data", {}).get("UpdateListItem")
+
+    if not result:
+        raise QueryException(resp['errors'])
+
+
+def check_item_nodes(item_ids: list):
+    """ Check if Items already exist in the CE
+
+    Arguments:
+        item_ids: The list of unique identifiers of the Item objects
+    Return:
+        Set of identifiers not found
+    """
+    query = query_listitems(identifiers=item_ids)
+    resp = submit_query(query)
+    result = resp.get("data", {}).get("ThingInterface")
+
+    not_found = set()
+    if not result:
+        raise QueryException(resp['errors'])
+    elif len(result) != len(item_ids):
+        not_found = set(item_ids) - set([item['identifier'] for item in result])
+
+    return not_found
+
+
+def check_itemlist_node(itemlist_id: str):
```

Comment:
Let's be careful about these two methods - they both start with `check_` but their behaviour is different. The previous one returns items that doesn't exist, and this one returns True/False.

For boolean checks, I often name things like `itemlist_node_exists`.

The previous method, we could call `get_nonexistent_listitem_nodes`, to make its purpose clear. Make sure you use 'listitem' instead of item to be explicit